packages/mobile/locales/en-US/goldEducation.json

```diff
@@ -2,15 +2,19 @@
   "steps":  [
     {
     "title": "What is CELO?",
-    "text": "CELO helps keep the Celo Dollar stable and the Celo Network running."
+    "text": "CELO helps keep the Celo Network running and the Celo Dollar stable."
     },
     {
       "title": "CELO’s price changes",
       "text": "Unlike Celo Dollars, the value of CELO changes. If more people use Celo Dollar, the value of CELO increases."
     },
     {
-      "title": "Please buy and sell responsibly ",
-      "text": "*Celo* allows you to buy and sell CELO. CELO presents a risk of financial loss so please do so responsibly. "
+      "title": "Shape the Celo Network",
+      "text": "By holding CELO, you have the ability to vote for changes and features you’d like to see on the Celo Network."
+    },
+    {
```

Comment:
This requires some changes in the code as well to be shown. @nityas can you please create a task for this and link this PR there?

Reply:
I just added strings for the events, would that be enough? If not I can remove-- wouldn't want to break the existing functionality